src/AST-Core/RBProgramNode.class.st

```diff
@@ -293,12 +293,11 @@ RBProgramNode >> do: aBlock [
 
 { #category : #generation }
 RBProgramNode >> dump [
-	^  String streamContents: [:s | self dumpOn: s]
-]
-
-{ #category : #generation }
-RBProgramNode >> dumpOn: aStream [
-	self subclassResponsibility 
+	"Generate the Pharo code from self"
```

Comment:
Generate a literal expression that recreates the receiver

Reply:
replaced.